src/components/plots/plot.ts

```diff
@@ -33,6 +33,7 @@ module Plottable {
     protected _renderArea: D3.Selection;
     protected _attrBindings: D3.Map<_Projection>;
     protected _attrExtents: D3.Map<any[]>;
+    protected _attrFilters: D3.Map<_Accessor>;
```

Comment:
How high in the hierarchy should filters go?  If it makes sense to exist here, then sure it should exist here.  However, if it should exist at the `XYPlot` level we should try to push to make sure it stays there